code/src/reducers/game.js

```diff
@@ -0,0 +1,66 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+import { ui } from './ui'
+
+const initialGame = {user:
+    { username: "" },
```

Comment:
what does it mean here with user and username? Is both necessary?

Reply:
Both is needed as we created a user object to be able to add more properties